eng/pipelines/templates/stages/archetype-python-release.yml

```diff
@@ -40,24 +41,26 @@ stages:
                           workingDirectory: $(Build.SourcesDirectory)
                           filePath: eng/scripts/SetTestPipelineVersion.ps1
                           arguments: '-BuildID $(Build.BuildId)'
+
                     - ${{if ne(artifact.skipVerifyChangeLog, 'true')}}:
                       - template: /eng/common/pipelines/templates/steps/verify-changelog.yml
                         parameters:
                           PackageName: ${{artifact.name}}
                           ServiceName: ${{parameters.ServiceDirectory}}
                           ForRelease: true
-                    - template: /eng/pipelines/templates/steps/stage-filtered-artifacts.yml
-                      parameters:
-                        SourceFolder: ${{parameters.ArtifactName}}
-                        TargetFolder: ${{artifact.safeName}}
-                        PackageName: ${{artifact.name}}
+
                     - pwsh: |
-                        Get-ChildItem -Recurse $(Pipeline.Workspace)/${{artifact.safeName}}
+                        $packageDirectory = "${{artifact.name}}".Replace("_", "-")
```

Comment:
Do we need to do this replacement? Can we just use the artifact name?

Reply:
Discussed offline. This is placed in lieu of touching every `ci.yml` in this PR. Will work with an admin to push the updating commit directly to master after this PR goes in.